raiden-ts/src/path/types.ts

```diff
@@ -68,3 +68,32 @@ export interface RaidenPFS {
   price: BigNumberish;
   token: string;
 }
+
+/**
+ * An IOU used to pay the services
+ */
+export const IOU = t.readonly(
+  t.type({
+    sender: Address,
+    receiver: Address,
+    amount: UInt(32),
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    expiration_block: UInt(32),
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    one_to_n_address: Address,
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    chain_id: UInt(32),
+  }),
+);
+
+// eslint-disable-next-line @typescript-eslint/interface-name-prefix
+export interface IOU extends t.TypeOf<typeof IOU> {}
+
+export const LastIOUResults = t.readonly(
+  t.type({
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    last_iou: IOU,
```

Comment:
Isn't this `Signed`? How to verify the last IOU returned by PFS is valid?

Reply:
Yes it is signed an yes we need to verify the signature